.travis/install-ivykis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x -e
+
+OLDPWD=`pwd`
+
+cd $TRAVIS_BUILD_DIR
```

Comment:
I like to quote every (string) variable for brevity (and syntax coloring), including this one and `--prefix=...`